packages/php/src/Metrics.php

```diff
@@ -80,7 +80,6 @@ public function __construct(public string $api_key, public string $group_handler
         ]);
 
         $this->package_version = InstalledVersions::getVersion(self::PACKAGE_NAME);
-        $this->cache_dir = Factory::createConfig()->get('cache-dir');
```

Comment:
Psalm doesn't like that`cache_dir` isn't being instantiated in the constructor anymore, mind silencing this?

```
Error: src/Metrics.php:32:20: PropertyNotSetInConstructor: Property ReadMe\Metrics::$cache_dir is not defined in constructor of ReadMe\Metrics or in any private or final methods called in the constructor (see https://psalm.dev/074)
```

Reply:
I instantiated `cache_dir` with null. This also makes the check if the cache_dir has been set or not solid as I'm not sure if the cache_dir can also be an empty string.